db/migrate/20170221141514_create_notification_log.rb

```diff
@@ -0,0 +1,19 @@
+class CreateNotificationLog < ActiveRecord::Migration
+  def change
+    create_table :notification_logs do |t|
```

Comment:
Perhaps we can set `null: false` for most/all of these fields so we won't need a lot of validation on the requests from `govuk_delivery`.